docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,82 @@
+```
+
+Since the children of relative layout are laid out depending on their relationship, you can create a responsive UI that keeps the proportions across different screen resolutions without modifications.
+width of `RectRect` and `BlueRect` are kept as `100` by their `WidthSpecification`. however, width of `GreenRect` and `YellowRect` are resized within layout space that is set by the `LeftTarget` and `RightTarget`:
+
+**Figure 2: Landscape mode of Figure 1**
+
+![relativeLayoutLandscapeExamplePreview](./media/relativeLayoutLandscapeExamplePreview.png)
+
+The following table describes the properties of `RelativeLayout`:
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `LeftTarget`           | View            | Gets or sets the left side edge of the child view relative to the given target view.|
+| `RightTarget`          | View            | Gets or sets the right side edge of the child view relative to the given target view.|
+| `TopTarget`            | View            | Gets or sets the top side edge of the child view relative to the given target view.|
+| `BottomTarget`         | View            | Gets or sets the bottom side edge of the child view relative to the given target view.|
+| `LeftRelativeOffset`   | float           | Gets or sets the relative offset for the left target. <br /> When the value is `0`, the left edges of the left target and child view are aligned.<br/> When the value is `1`, the left edge of the child view is aligned to the right edge of the left target. |
+| `RightRelativeOffset`  | float           | Gets or sets the relative offset for the right target. <br /> When the value is `0`, the right edge of the child view is aligned to the left edge of the right target.<br/> When the value is `1`, the right edges of the right target and child view are aligned.
+| `TopRelativeOffset`    | float           | Gets or sets the relative offset for the top target. <br /> When the value is `0`, the top edges of the top target and child view are aligned.<br/> When the value is `1`, the top edge of the child view is aligned to the bottom edge of the top target.
+| `BottomRelativeOffset` | float           | Gets or sets the relative offset for the bottom target. <br /> When the value is `0`, the bottom edge of the child view is aligned to the top edge of the bottom target.<br/>When the value is `1`, the bottom edges of the bottom target and child view are aligned.
+| `HorizontalAlignment`  | RelativeLayout.Alignment| Gets or sets the horizontal alignment of the child view.|
+| `VerticalAlignment`  | RelativeLayout.Alignment| Gets or sets the vertical alignment of the child view.|
+| `FillHorizontal`  | bool | Gets or sets the boolean value indicating whether the child fills its horizontal space or not.|
+| `FillVertical`  | bool | Gets or sets the boolean value indicating whether the child fills its vertical space or not.|
+
+## Target and relativeOffset
```

Comment:
I understand these are properties but can we just write the name normally.

Reply:
It changed to 
## Layout space of the child view